translate/filters/test_checks.py

```diff
@@ -1365,23 +1365,160 @@ def test_skip_checks_per_language_in_some_checkers():
     """Test some checks are skipped for some languages in Mozilla checker."""
     from translate.storage import base
 
+    # Hijack checker config language ignoretests to test check is skipped.
+    checker_config = checks.CheckerConfig(targetlanguage="gl")
+    previous_ignoretests = checker_config.lang.ignoretests
+    checker_config.lang.ignoretests = {
+        'mozilla': ['accelerators'],
+    }
+
+    # Prepare the checkers and the unit.
+    mozillachecker = checks.MozillaChecker(checkerconfig=checker_config)
+    stdchecker = checks.StandardChecker(
+        checkerconfig=checks.CheckerConfig(accelmarkers="&",
+                                           targetlanguage="as")
+    )
+
     str1, str2, __ = strprep(u"&Check for updates", u"আপডেইটসমূহৰ বাবে নিৰীক্ষণ কৰক")
     unit = base.TranslationUnit(str1)
     unit.target = str2
 
+    # Accelerators check is disabled for Assamese in MozillaChecker.
+    assert 'accelerators' not in mozillachecker.run_filters(unit).keys()
```

Comment:
mozillachecker == 'gl' as far as I can see. What is the mention of Assamse here?